airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2125,7 +2125,7 @@ components:
         - append
         - overwrite
         #- upsert_dedup # TODO chris: SCD Type 1 can be implemented later
-        - append_dedup # SCD Type 2
+        - append_dedup # SCD Type 1 & 2
```

Comment:
why is 1 and 2 now? should the comment above be removed then?

Reply:
is there not a field in this file that needs to set sync mode to required? or is it already set?

Reply:
append_dedup produces two tables:
- history table (SCD type 2)
- dedup table (SCD type 1)
So I just made the comment truthier

upsert_dedup without the history table would use less space (since no SCD type 2 would be stored)